packages/amount-selectors/src/components/AmountSelectors.test.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import AmountSelectors from "./AmountSelectors";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("Amount selectors", () => {
+  describe("Buttons", () => {
+    test("decrement button should decrease the state value by 1", () => {
+      const initialValue = 2;
+      const expectedValue = 1;
+      const component = shallow(<AmountSelectors value={initialValue} />);
+      component
+        .children("Button")
+        .at(0)
```

Comment:
Can't we use a more descriptive selector? Getting the first position of an arrays seems weird. It works, but what if the children order changes at some point?